cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -171,13 +171,6 @@ func runApply(flags *applyFlags, args []string) error {
 		return err
 	}
 
-	// TODO: https://github.com/kubernetes/kubeadm/issues/2375
-	fmt.Printf("[upgrade/postupgrade] Applying label %s='' to control plane Nodes\n",
-		kubeadmconstants.LabelExcludeFromExternalLB)
-	if err := upgrade.LabelControlPlaneNodesWithExcludeFromLB(client); err != nil {
```

Comment:
IMO, we can completely  remove `LabelControlPlaneNodesWithExcludeFromLB` function as it is only called here.

Ref: https://github.com/kubernetes/kubernetes/blob/a55bd631728590045b51a4f65bba31aed1415571/cmd/kubeadm/app/phases/upgrade/postupgrade.go#L234-L260